src/pages/SigningOfficialConsole.js

```diff
@@ -73,13 +81,35 @@ export default function SigningOfficialConsole(props) {
               href: 'https://broad-duos.zendesk.com/hc/en-us/articles/360060402751-Signing-Official-User-Guide',
               target: '_blank',
               id: 'so-console-info-link'},
-            ['Click Here for more information'])
+            ['Click Here for more information']),
+
           ])
         ])
       ]),
       div({style: {borderTop: '1px solid #BABEC1', height: 0}}, []),
-      //researcher table goes here
-      h(SigningOfficialTable, {researchers, signingOfficial, unregisteredResearchers, isLoading}, []),
+      div({style: {}, class: 'signing-official-tabs'}, [
+        //NOTE: placeholder styling for now, can come up with more definitive designs later
+        div({style: {display: 'flex'}, class: 'tab-selection-container'}, [
+          h(SelectableText, {
+            label: 'Data Custodian',
+            color: 'red',
+            fontSize: `1.8rem`,
+            componentType: tabs.custodian,
+            setSelected: setSelectedTag,
+            selectedType: selectedTag
+          }),
+          h(SelectableText, {
+            label: 'Researchers',
+            color: 'blue',
+            fontSize: '1.8rem',
+            componentType: tabs.researcher,
+            setSelected: setSelectedTag,
+            selectedType: selectedTag
+          })
+        ]),
+        h(SigningOfficialTable, {isRendered: selectedTag === tabs.researcher, researchers, signingOfficial, unregisteredResearchers, isLoading}, []),
+        h(DataCustodianTable, {isRendered: selectedTag === tabs.custodian, researchers, signingOfficial, unregisteredResearchers, isLoading}, [])
+      ]),
```

Comment:
Can we wrap the new content in a conditional on env? Look at https://github.com/DataBiosphere/duos-ui/blob/develop/src/libs/config.js#L6 which should give you the correct environment. See also:
* https://duos-k8s.dsde-dev.broadinstitute.org/config.json
* https://duos-k8s.dsde-staging.broadinstitute.org/config.json
* https://duos-k8s.dsde-prod.broadinstitute.org/config.json

Reply:
What's the conditional? Don't render on production?

Reply:
Yes, exactly. Staging and dev should be fine.